dataservice/api/common/custom_fields.py

```diff
@@ -95,3 +96,19 @@ def _deserialize(self, value, attr, data):
             raise ValidationError('Invalid date or datetime string')
 
         return value
+
+
+class EnumColumn(fields.Field):
+    """Validates against a given set of enumerated values."""
+
+    def __init__(self, enum, *args, **kwargs):
```

Comment:
You always pass a `list` as enum here.
Why not just pass in an enum natively instead?
So this:
```
EnumColumn(enum=[s.value for s in RaceEnum])
```
Becomes:
```
EnumColumn(enum=RaceEnum)
```
Then you take care of the list comprehension inside the `_validate` function